prime.cpp

```diff
@@ -0,0 +1,26 @@
+#include<iostream>
+using namespace std;
+bool prime(int n){
+    if(n==1){
+        return false;
+    }
+    if(n==2){
+        return true;
+    }
+    int temp=0;
```

Comment:
Can we do it without using temp variable.

Reply:
yes i can do